jacorb-name-server/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM jboss/base:latest
+MAINTAINER Gytis Trikleris <gytis@redhat.com>
+WORKDIR /opt/jboss
+EXPOSE 3528
+
+ENV JACORB_VERSION=2.3.1 \
+    JACORB_HOME=/opt/jboss/jacorb \
+    NAME_SERVER_PORT=3528 \
+    PATH=$PATH:/opt/jboss/jacorb/bin
+
+USER root
+
+RUN yum install -y ant \
+    && yum clean all
+
+USER jboss
+
+RUN curl -LOk http://www.jacorb.org/releases/$JACORB_VERSION/jacorb-$JACORB_VERSION-bin.zip \
```

Comment:
Builds are scheduled. I have troubles with setting proper version tags to these images, since I have no idea what it could be right now. But `latest` will be available soon.